### ISSUE TITLE
is U's

### ISSUE THREAD

**Issue by JTLiggett** [LunaC25/csPortfolio#2]
I liked the color choices on your webpage. Maybe you could make the text bigger and add more dogs to make it butter.
The use of mark down is pretty cool. Maybe adding images would make it cooler.
The chemotaxis was clearly unique. How did you get the weird shapes to appear? Also, those dots are green, not blue.

**Comment by bartzc** [comment]
I agree that the mark down would be improved if there were images. Also agree how all the projects are unique.
<details><summary><strong>Quotes</strong></summary>
    <p>

  >Dwight: "We have too many people on this earth. We need a new plague."
    </p>
  </details>
